doc/articles/features/windows-storage.md

```diff
@@ -0,0 +1,28 @@
+# Windows.Storage
+
+Uno supports some of the APIs from the `Windows.Storage` namespace, such as `Windows.Storage.StorageFile` and `Windows.Storage.StorageFolder` for all platforms.
+
+Both `Windows.Storage` and `System.IO` APIs are available, with some platform specifics defined below. In general, it is best to prefer the use of `Windows.Storage` APIs when available, as their async nature allows for transparent interactions with the underlying file system implementations.
+
+## WebAssembly support
+
+WebAssembly file system APIs are built using [emscripten's POSIX file system APIs](https://emscripten.org/docs/api_reference/Filesystem-API.html). The persistence is done through the use of browser APIs, such as IndexedDB through [emscripten's IDBFS](https://emscripten.org/docs/api_reference/Filesystem-API.html#filesystem-api-idbfs).
+
+While it is possible to write files in any paths, only some folders are persisted across browser refreshes:
+- `ApplicationData.Current.LocalFolder`
+- `ApplicationData.Current.RoamingFolder`
+- `ApplicationData.Current.SharedLocalFolder`
+
+Note that the initialization of the filesystem is asynchronous. This means that reading a file during `Application.OnLaunched` using `File.OpenRead` may fail to find a file that was previously written.
```

Comment:
```suggestion
Note that the initialization of the filesystem is asynchronous. This means that reading a file during `Application.OnLaunched` using `System.IO.File.OpenRead` may fail to find a file that was previously written, because the filesystem is not available yet.
```